workspaces/quickstart/app-config.yaml

```
@@ -3,62 +3,86 @@ app:
   baseUrl: http://localhost:3000
   quickstart:
     - title: Set up authentication
+      titleKey: steps.title.setupAuthentication
```

I think its better to group these by their step instead of grouping all titles. This gives the translation team more context:

```suggestion
      titleKey: steps.setupAuthentication.title
```